src/phpcs/Production/Sniffs/Commenting/MethodDocBlockHasDescriptionSniff.php

```diff
@@ -56,6 +56,10 @@ private function hasMethodDocBlockDescription(PHP_CodeSniffer_File $sniffedFile,
      * @param int                  $index position of current token in token list
      *
      * @return bool
+     *
+     * @throws PHP_CodeSniffer_Exception If the specified token is not of type
```

Comment:
There is no all of "throw". No Annotation needed

Reply:
Agreed.